metrics/README.md

```diff
@@ -1,5 +1,31 @@
 # Kata Containers metrics
 
+* [Kata Containers metrics](#kata-containers-metrics)
+   * [Goals](#goals)
+      * [PR regression checks](#pr-regression-checks)
+      * [master branch trending](#master-branch-trending)
+      * [Developer pre-checking](#developer-pre-checking)
+   * [Stability or Performance?](#stability-or-performance)
+   * [Requirements](#requirements)
+      * [For PR checks](#for-pr-checks)
+      * [For master tracking](#for-master-tracking)
+   * [Categories](#categories)
+      * [Time (Speed)](#time-speed)
+      * [Density](#density)
+      * [Networking](#networking)
+      * [Storage](#storage)
+   * [Saving Results](#saving-results)
+      * [JSON API](#json-api)
+         * [metrics_json_init()](#metrics_json_init)
+         * [metrics_json_save()](#metrics_json_save)
+         * [metrics_json_add_fragment(json)](#metrics_json_add_fragmentjson)
+         * [metrics_json_start_array()](#metrics_json_start_array)
+         * [metrics_json_add_array_element(json)](#metrics_json_add_array_elementjson)
+         * [metrics_json_add_array_fragment(json)](#metrics_json_add_array_fragmentjson)
+         * [metrics_json_close_array_element()](#metrics_json_close_array_element)
+         * [metrics_json_end_array(name)](#metrics_json_end_arrayname)
+   * [Preserving results Results](#preserving-results)
```

Comment:
Line 27: I'm assuming the middle instance of "results" isn't needed.

Reply:
Indeed. fixed.